contracts/VeERC20.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/utils/Context.sol";
+
+import "./interfaces/IVeERC20.sol";
+
+/// @title VeERC20
+/// @notice Modified version of ERC20 where transfers and allowances are disabled.
+/// @dev Only minting and burning are allowed. The hook `_beforeTokenOperation` and
+/// `_afterTokenOperation` methods are called before and after minting/burning respectively.
+contract VeERC20 is Context, IVeERC20 {
+    mapping(address => uint256) private _balances;
+
+    uint256 private _totalSupply;
+
+    string private _name;
+    string private _symbol;
+
+    /// @dev Emitted when `value` tokens are burned and minted
+    event Burn(address indexed account, uint256 value);
+    event Mint(address indexed beneficiary, uint256 value);
+
+    /**
+     * @dev Sets the values for {name} and {symbol}.
```

Comment:
Above you use backticks when referencing a variable and here curly brackets, is there a reason for this (maybe renders a different way?), if not would prefer to stick to one style.

Reply:
Yeah this was mainly copied from [`ERC20.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol) which seems to mix both